### ISSUE TITLE
update via 'brew upgrade' fails

### ISSUE THREAD

**Issue by caponecicero** [apparition47/MailTrackerBlocker#85]
...and the cask didn't install.  Here's the deets:
```
stevewi:~ $ brew upgrade
Updating Homebrew...
==> Auto-updated Homebrew!
Updated 3 taps (apparition47/tap, homebrew/core and homebrew/cask).
==> New Formulae
virtualenv
==> Updated Formulae
Updated 71 formulae.
==> Renamed Formulae
kde-extra-cmake-modules -> extra-cmake-modules
kde-karchive -> karchive
kde-kdoctools -> kdoctools
kde-ki18n -> ki18n
kde-threadweaver -> threadweaver
==> Updated Casks
Updated 394 casks.

==> Casks with 'auto_updates' or 'version :latest' will not be upgraded; pass `-
==> Upgrading 1 outdated package:
apparition47/tap/mailtrackerblocker 0.3.21 -> 0.3.24
==> Upgrading mailtrackerblocker
==> Downloading https://github.com/apparition47/MailTrackerBlocker/releases/down
==> Downloading from https://github-releases.githubusercontent.com/277057483/4eb
######################################################################## 100.0%
==> Uninstalling packages:
com.onefatgiraffe.mailtrackerblocker
Password:
find: /private/tmp/MailTrackerBlocker-Installation-Temp: No such file or directory
==> Purging files for version 0.3.24 of Cask mailtrackerblocker
Error: apparition47/tap/mailtrackerblocker: Failure while executing; `/usr/bin/sudo -E -- /usr/bin/xargs -0 -- /bin/bash -c set\ -euo\ pipefail'
''
'for\ path\ in\ \"\$\{@\}\"\;\ do'
'\ \ if\ \[\[\ -e\ \"\$\{path\}/.DS_Store\"\ \]\]\;\ then'
'\ \ \ \ /bin/rm\ -f\ \"\$\{path\}/.DS_Store\"'
'\ \ fi'
''
'\ \ \#\ Some\ packages\ leave\ broken\ symlinks\ around\;\ we\ clean\ them\ out\ before'
'\ \ \#\ attempting\ to\ \`rmdir\`\ to\ prevent\ extra\ cruft\ from\ accumulating.'
'\ \ /usr/bin/find\ \"\$\{path\}\"\ -mindepth\ 1\ -maxdepth\ 1\ -type\ l\ \!\ -exec\ /bin/test\ -e\ \{\}\ \\\;\ -delete'
''
'\ \ if\ \[\[\ -L\ \"\$\{path\}\"\ \]\]\;\ then'
'\ \ \ \ \#\ Delete\ directory\ symlink.'
'\ \ \ \ /bin/rm\ \"\$\{path\}\"'
'\ \ elif\ \[\[\ -d\ \"\$\{path\}\"\ \]\]\;\ then'
'\ \ \ \ \#\ Delete\ directory\ if\ empty.'
'\ \ \ \ /usr/bin/find\ \"\$\{path\}\"\ -maxdepth\ 0\ -type\ d\ -empty\ -exec\ /bin/rmdir\ \{\}\ \\\;'
'\ \ else'
'\ \ \ \ \#\ Try\ \`rmdir\`\ anyways\ to\ show\ a\ proper\ error.'
'\ \ \ \ /bin/rmdir\ \"\$\{path\}\"'
'\ \ fi'
'done'
' --` exited with 1. Here's the output:
find: /private/tmp/MailTrackerBlocker-Installation-Temp: No such file or directory
```

![About](https://user-images.githubusercontent.com/2354165/111909240-c2af0e80-8a19-11eb-9eaa-6e94205a5ec3.jpg)

**Comment by massimobio** [comment]
Same here. 
Also, `brew uninstall mailtrackerblocker` fails with the same error.

**Comment by apparition47** [comment]
Please see comment here for a fix (also see the official Cask migration comment below as well): apparition47/homebrew-tap#1

This issue should be fixed starting 0.3.22.

**Comment by massimobio** [comment]
I should have searched closed issues. Thank you.
